app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Admin::DashboardController < Admin::BaseController
-    def index
+  def index
+    @orders = Order.all
```

Comment:
Wondering if we can just do a model method here that gives us the whole collection sorted how we want. Might be something to look into